worker/rhpv3.go

```diff
@@ -41,15 +41,15 @@ func (w *worker) fundAccount(ctx context.Context, account *account, pt rhpv3.Hos
 	})
 }
 
-func (w *worker) syncAccount(ctx context.Context, account *account, pt rhpv3.HostPriceTable, siamuxAddr string, hostKey types.PublicKey) error {
+func (w *worker) syncAccount(ctx context.Context, account *account, pt rhpv3.HostPriceTable, hostIP string, hostKey types.PublicKey) error {
 	account, err := w.accounts.ForHost(hostKey)
 	if err != nil {
 		return err
 	}
 	payment := w.preparePayment(hostKey, pt.AccountBalanceCost, pt.HostBlockHeight)
 	return account.WithSync(ctx, func() (types.Currency, error) {
 		var balance types.Currency
-		err := withTransportV3(ctx, siamuxAddr, hostKey, func(t *rhpv3.Transport) error {
+		err := withTransportV3(ctx, hostIP, hostKey, func(t *rhpv3.Transport) error {
```

Comment:
I'm not sure what is going on with the `hostIP` and `siamuxAddr` lately but it's been biting me left and right as well 😁